Web/Pages/CCGMember/Compare/Delta.cshtml

```diff
@@ -0,0 +1,269 @@
+                                <td width="50%">@(isEnglish?matchingtype2[i].CompetencyNameEng:matchingtype2[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                        else
+                        {
+                            <tr>
+                                <td width="50%">@null</td>
+                                <td width="50%">@(isEnglish?matchingtype2[i].CompetencyNameEng:matchingtype2[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                    }
+                }
+                else
+                {
+                    @for (int i = 0; i < matchingcertificates.Length; i++)
+                    {
+                        if (i < matchingtype2.Length)
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?matchingcertificates[i].NameEng:matchingcertificates[i].NameFre)</td>
+                                <td width="50%">@(isEnglish?matchingtype2[i].CompetencyNameEng:matchingtype2[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                        else
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?matchingcertificates[i].NameEng:matchingcertificates[i].NameFre)</td>
+                                <td width="50%">@null</td>
+                            </tr>
+                        }
+                    }
+                }
+                <tr>
+                    <td width="50%"></td>
+                    <td width="50%"></td>
+                </tr>
+                <tr>
+                    <th style="background-color:#f2f2f2" width="50%">@Localizer["KNOWLEDGE ELEMENTS"]</th>
+                    <th style="background-color:#f2f2f2" width="50%">@Localizer["BEHAVIOURAL ELEMENTS"]</th>
+                </tr>
+                @if (matchingtype1.Length >= matchingtype3.Length)
+                {
+                    @for (int i = 0; i < matchingtype1.Length; i++)
+                    {
+                        if (i < matchingtype3.Length)
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?matchingtype1[i].CompetencyNameEng:matchingtype1[i].CompetencyNameFre)</td>
+                                <td width="50%">@(isEnglish?matchingtype3[i].CompetencyNameEng:matchingtype3[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                        else
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?matchingtype1[i].CompetencyNameEng:matchingtype1[i].CompetencyNameFre)</td>
+                                <td width="50%">@null</td>
+                            </tr>
+                        }
+                    }
+                    <tr>
```

Comment:
why is there an empty row? table markup should be used for styling presentation.